back/engines/free/email_campaigns/app/mailers/email_campaigns/event_registration_confirmation_mailer.rb

```diff
@@ -32,8 +32,11 @@ def event_title
     end
 
     def event_time
-      start_at = I18n.l(event.event_attributes.start_at, format: :short, locale: locale)
-      end_at = I18n.l(event.event_attributes.end_at, format: :short, locale: locale)
+      timezone_name = AppConfiguration.instance.settings.dig('core', 'timezone')
+      timezone = ActiveSupport::TimeZone[timezone_name] || (raise KeyError, timezone_name)
+
```

Comment:
used the `.in_time_zone()` method which was already in use in the .ics generator - which was returning the correct dates